components/NavigationBar/index.js

```diff
@@ -60,7 +66,7 @@ export default function NavigationBar() {
     <Navbar>
       <NavLink href="/"><HomeIcon/></NavLink>
       {session ? (
-        <NavLink href="/new-project"><PlusIcon/></NavLink>
+        <NavLink href="/new-project" className="plusbutton"><PlusIcon/></NavLink>
```

Comment:
You might want to consider not using `className` since we traditionally don't use it. Maybe another styled component for the icon would be better.